pkg/fs/fs.go

```diff
@@ -762,7 +763,9 @@ func (fs *FileSystem) doResolve(ctx meta.Context, p string, followLastSymlink bo
 		if len(name) == 0 {
 			continue
 		}
-		if parent == meta.RootInode && i == len(ss)-1 && vfs.IsSpecialName(name) {
+		isSpCtrlFile := i == len(ss)-1 && vfs.IsSpecialControlName(name)
```

Comment:
We don't want to expose this to Java SDK and S3 gateway, so it's better to have this only for winfs

Reply:
changes were made, now it only affects the behavior on windows.